test/chemistry/test_qeom_ee.py

```diff
@@ -35,11 +38,13 @@ def setUp(self):
             pyscf_driver = PySCFDriver(atom=atom,
                                        unit=UnitsType.ANGSTROM, charge=0, spin=0, basis='sto3g')
             self.molecule = pyscf_driver.run()
+            warnings.filterwarnings('ignore', category=DeprecationWarning)
             core = Hamiltonian(transformation=TransformationType.FULL,
```

Comment:
Again the goal was to test EomEE - the components used to do so can be switched out to newer ones. Having said that EomEE will be deprecated too of course...

Reply:
this will come with the new Excited States PR